snap/bin/cassandra-wrapper.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash -e
+
+# this script combined from https://github.com/snapcrafters/cassandra/blob/3df209fafb7b3dd341ec7412be4725d7104c5a54/common
+# and from https://github.com/snapcrafters/cassandra/blob/3df209fafb7b3dd341ec7412be4725d7104c5a54/wrapper-cassandra
+
+if [ -z "$SNAP_COMMON" ]; then
+    # https://github.com/snapcore/snapd/commit/da9fa37
```

Comment:
Why is there here? The commit you reference looks like it's the original snapd commit which adds $SNAP_COMMON. Is this a guard against a system running an older version of snapd which doesn't export $SNAP_COMMON?

Reply:
This was copied from the original cassandra snap that snapcrafters made : https://github.com/snapcrafters/cassandra/blob/3df209fafb7b3dd341ec7412be4725d7104c5a54/common#L2

I can modify the comments if you like to not reference this, but yes I believe this is to guard against a system running an older version of snapd which doesn't export `$SNAP_COMMON`